update-verifier/src/lib.rs

```diff
@@ -21,13 +22,14 @@ pub const BUILD_INFO: BuildInfo = make_build_info!();
 pub fn run_health_check() -> eyre::Result<()> {
     // get runtime environment variable to force health check
     let dry_run = std::env::var("UPDATE_VERIFIER_DRY_RUN").is_ok();
+    let orb_slot_ctrl = OrbSlotCtrl::new(&EfiVarDb::from_rootfs("/")?)?;
```

Comment:
I know we already went over this, but I'll admit I'm still skeptical of dependency injection _even in application code_ as the answer to testability.

We looked a bit together, and I'll admit that alternatives besides doing something a bit more hack-y don't immediately jump out at me here. But I think in general testing should be _**in service of**_ application code.

Reply:
I thought more about this, and I think the issue I have with this testing is that it's testing `File::open` and `File::read`. That test, applied to the `Fixture` here, does not match the real-world scenario of reading and writing to the `efifvars`.